TYPO3.Neos.Kickstarter/Classes/Service/GeneratorService.php

```diff
@@ -45,11 +42,16 @@ class GeneratorService extends \TYPO3\Kickstart\Service\GeneratorService
      */
     public function generateSitePackage($packageKey, $siteName)
     {
-        $packageMetaData = new MetaData($packageKey);
-        $packageMetaData->addConstraint(new PackageConstraint(MetaDataInterface::CONSTRAINT_TYPE_DEPENDS, 'TYPO3.Neos'));
-        $packageMetaData->addConstraint(new PackageConstraint(MetaDataInterface::CONSTRAINT_TYPE_DEPENDS, 'TYPO3.Neos.NodeTypes'));
-        $packageMetaData->addConstraint(new PackageConstraint(MetaDataInterface::CONSTRAINT_TYPE_SUGGESTS, 'TYPO3.Neos.Seo'));
-        $this->packageManager->createPackage($packageKey, $packageMetaData, null, 'typo3-flow-site');
+        $this->packageManager->createPackage($packageKey, [
+            'type' => 'typo3-flow-site',
```

Comment:
should be `neos-site` with https://github.com/neos/neos-development-collection/pull/1242 in mind